bin/crew

```diff
@@ -983,11 +983,54 @@ def prepare_package(destdir)
 
     strip_dir destdir
 
+    # Use patchelf to set need paths for all binaries
+    # currently broken due to glibc issue on armv7l
+    unless ARCH == 'armv7l' || ARCH == 'aarch64'
+       patchelf_set_need_paths destdir
+    end
+
     # use upx on executables
     shrink_dir destdir
   end
 end
 
+def patchelf_set_need_paths(dir)
+  return unless @pkg.no_patchelf.to_s.empty?
```

Comment:
`@pkg.no_patchelf` should be already a boolean?

Reply:
It is `nil` if it doesn't exist. There is code in `package.rb` to make `is_fake` a Boolean, but I think we can avoid having to do that for everything if just use the `.to_s.empty?` trick!

Reply:
As I know `nil` acts like `false` in comparing (`if nil` is the same as `if false`)?

As an alternative, we can use `!!@pkg.no_patchelf` to force convert the value to boolean (All object except `nil` and `false` will convert to `true`, `nil` and `false` will convert to `false`)

Reply:
Unfortunately, environment variables don't work this way.  `ENV['SOME_VARIABLE']=false` is actually true.  In order to make an environment variable "false", it would need to be unset like `ENV['SOME_VARIABLE']=`.

Reply:
If this works we should use if everywhere...

Reply:
@supechicken I can't seem to get this to work. `!!@pkg.no_patchelf` is always false. In fact, the other properties I set aren't showing up either. Do we need to be modifying package_helpers to make sure that properties are set?

What's the simplest way to put a property in a file and then be able to see if it is there?

And why are we setting all these properties in package_helpers?

Should we be using `include DataMapper::Resource` like [here](https://www.rubydoc.info/github/datamapper/dm-core/DataMapper/Property)?